components/user_settings/notifications/desktop_notification_settings.jsx

```diff
@@ -49,6 +79,32 @@ export default class DesktopNotificationSettings extends React.PureComponent {
                 soundRadio[0] = true;
             }
 
+            if (this.props.sound === 'true') {
+                const sounds = Array.from(Utils.notificationSounds.keys());
+                const options = sounds.map((sound) => {
+                    return {value: sound, label: sound};
+                });
+
+                let allowNotificationSelection = true;
+                if (isDesktopApp() && window.desktop && semver.lte(window.desktop.version, '4.6.0')) {
+                    allowNotificationSelection = false;
+                }
+
+                if (allowNotificationSelection) {
```

Comment:
Again, if we add this feature on 4.6 we shouldn't exclude it. Also, we can simplify a bit here.
```suggestion
                if (!isDesktopApp() || (window.desktop && semver.gte(window.desktop.version, '4.6.0'))) {
```